Library/src/main/java/info/metadude/android/typedpreferences/BasePreference.java

```diff
@@ -0,0 +1,33 @@
+package info.metadude.android.typedpreferences;
+
+import android.content.SharedPreferences;
+
+/**
+ * A wrapper class for a boolean preference.
```

Comment:
Please clean up the comments, `An abstract base class for preference.` might be fitting.